lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -60,6 +60,14 @@ UsdUndoRenameCommand::UsdUndoRenameCommand(const UsdSceneItem::Ptr& srcItem, con
 {
     const UsdPrim& prim = _stage->GetPrimAtPath(_ufeSrcItem->prim().GetPath());
 
+    // eraly check to see if a particular node has any specs to contribute
+    // to the final composed prim.
```

Comment:
Would be good to mention how that can happen (e.g. reference).

Reply:
addressed in 7fafef624444489ef3ddddd200706b9a57ed353c